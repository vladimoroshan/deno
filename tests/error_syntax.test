args: tests/error_syntax.js --reload
check_stderr: true
exit_code: 1
output: tests/error_syntax.js.out
