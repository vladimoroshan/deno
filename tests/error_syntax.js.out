error: Uncaught SyntaxError: Unexpected identifier
[WILDCARD]tests/error_syntax.js:3:6

3 (the following is a syntax error  ^^ ! )
       ~~~~~~~~~

